make/RunTestsPrebuilt.gmk

```diff
@@ -170,6 +170,8 @@ else
     OPENJDK_TARGET_OS := linux
   else ifeq ($(UNAME_OS), Darwin)
     OPENJDK_TARGET_OS := macosx
+  else ifneq ($(findstring BSD,$(UNAME_OS)),)
```

Comment:
Please always use a space after comma, if possible. (It is possible in most make expressions; the most common exception is `patsubst`).

Reply:
Agreed!